controllers/openstackcluster_controller.go

```diff
@@ -333,7 +333,13 @@ func reconcileBastion(log logr.Logger, osProviderClient *gophercloud.ProviderCli
 		handleUpdateOSCError(openStackCluster, errors.Errorf("failed to get or create floating IP for bastion: %v", err))
 		return errors.Errorf("failed to get or create floating IP for bastion: %v", err)
 	}
-	err = computeService.AssociateFloatingIP(instance.ID, fp.FloatingIP)
+	port, err := computeService.GetControlPlanePort(instance)
+	if err != nil {
+		err = errors.Errorf("getting control plane port for bastion: %v", err)
```

Comment:
This port is not control plane port, so should be "getting port for bastion: %v".